modules/violation-view-policy-tab.adoc

```diff
@@ -9,21 +9,34 @@
 The *Policy* tab of the *Details* panel displays details of the policy that caused the violation.
 
 [discrete]
-== Policy Details section
+== Policy overview section
 
-The policy details section lists the following information:
+The policy overview section lists the following information:
 
-* *Id*: The numerical identifier for the policy.
-* *Name*: The name of the policy.
+* *Severity* - The risk level for the violation.
+* *Categories*: The policy category of the policy.
+* *Type*:
 * *Description*: A detailed explanation of what the policy alert is about.
 * *Rationale*: Information about the reasoning behind the establishment of the policy and why it matters.
-* *Remediation*: Suggestions on how to fix the violation.
-* *Enabled*: Indicates if the policy is enabled.
-* *Categories*: The policy category of the policy.
+* *Guidance*: Suggestions on how to address the violation.
+* *MITRE ATT&CK*: Indicates whether or not the policy has MITRE ATT&CK vectors.
+
+//These no longer seem to be in the GUI:
+//* *Id*: The numerical identifier for the policy.
+//* *Name*: The name of the policy.
+//* *Enabled*: Indicates if the policy is enabled.
+
+[discrete]
+== Policy behavior
+
+The policy behavior section provides the following information:
+
 * *Lifecycle Stage*: Lifecycle stages that the policy belongs to, `Build`, `Deploy`, or `Runtime`.
-* *Severity* - The risk level for the violation.
+* *Event source*:
+* *Response*:
+* *Enforcement*:
```

Comment:
If the response is set to Inform and enforce, then this shows what type of enforcement is set (build, deploy, runtime).

From existing docs:

```
Build - RHACS fails your continuous integration (CI) builds when images match the criteria of the policy.

Deploy - RHACS blocks creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or OpenShift Container Platform API server blocks all noncompliant deployments. In other clusters, RHACS edits noncompliant deployments to prevent pods from being scheduled.

Runtime - RHACS deletes all pods when an event in the pods matches the criteria of the policy.
```